test/lib/Benchmark.js

```diff
@@ -299,4 +301,15 @@ Benchmark.prototype = {
   })
 };
 
+const PREF_LOG_LEVEL = `extensions.${self.id}.sdk.console.logLevel`;
+
+// forceLog allows the user to use the console.log() regardless the current log level
+function forceLog(...theArgs) {
+  let logLevel = prefs.get(PREF_LOG_LEVEL, "error");
+  prefs.set(PREF_LOG_LEVEL, "info");
+  console.log.apply(console, theArgs); // eslint-disable-line
```

Comment:
you could alternatively just do `console.log.call(console, ...arguments)` instead of using `theArgs`